pkg/apis/serving/v1/configuration_defaults_test.go

```diff
@@ -156,6 +157,59 @@ func TestConfigurationDefaulting(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "run latest with identical timeout defaults",
+		in: &Configuration{
+			Spec: ConfigurationSpec{
+				Template: RevisionTemplateSpec{
+					Spec: RevisionSpec{
+						PodSpec: corev1.PodSpec{
+							EnableServiceLinks: ptr.Bool(true),
+							Containers: []corev1.Container{{
+								Image: "busybox",
+							}},
+						},
+						ContainerConcurrency: ptr.Int64(config.DefaultContainerConcurrency),
+					},
+				},
+			},
+		},
+		want: &Configuration{
+			Spec: ConfigurationSpec{
+				Template: RevisionTemplateSpec{
+					Spec: RevisionSpec{
+						PodSpec: corev1.PodSpec{
+							EnableServiceLinks: ptr.Bool(true),
+							Containers: []corev1.Container{{
+								Name:           config.DefaultUserContainerName,
+								Image:          "busybox",
+								Resources:      defaultResources,
+								ReadinessProbe: defaultProbe,
+							}},
+						},
+						TimeoutSeconds:       ptr.Int64(423),
+						ContainerConcurrency: ptr.Int64(config.DefaultContainerConcurrency),
+					},
+				},
+			},
+		},
+		ctx: func() context.Context {
+			logger := logtesting.TestLogger(t)
+			s := cconfig.NewStore(logger)
+			s.OnConfigChanged(&corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: config.FeaturesConfigName}})
+			s.OnConfigChanged(&corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: config.DefaultsConfigName,
+				},
+				Data: map[string]string{
+					"revision-timeout-seconds":                "423",
```

Comment:
Why was `423` chosen as the timeout value? Consider using a named constant or variable to make the
intent clear and maintain consistency across test files.

Reply:
Just picked a number. I will update with a constant.